src/zenflows/gql/mw/debug.ex

```diff
@@ -0,0 +1,38 @@
+# Zenflows is designed to implement the Valueflows vocabulary,
+# written and maintained by srfsh <info@dyne.org>.
+# Copyright (C) 2021-2022 Dyne.org foundation <foundation@dyne.org>.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+defmodule Zenflows.GQL.MW.Debug do
+@moduledoc """
+Absinthe middleware to verify GraphQL calls.
+"""
+
+@behaviour Absinthe.Middleware
+
+@impl true
+def call(res, _opts) do
+	if res.context[:authenticate_calls?] do
```

Comment:
Use `Map.has_key?(res.context, :authenticate_calls?)` here, since we don't want to deal with the `nil` case.

Reply:
What do you mean "we don't want to deal with the nil case"?
If I run `x=%{a: nil}` and then `Map.has_key?(x, :a)` it evaluates to true (and so we have introduced a nil inside `res.context`)
On the other hand, without `Map.has_key?` it could be both nil and absent, I'll always look at the conf